setup.py

```diff
@@ -3,14 +3,16 @@
 import setuptools
 from pathlib import Path
 
+project_dir = Path(__file__).parent
```

Comment:
In the past I've needed to use [`Path(__file__).resolve().parent`](https://docs.python.org/3/library/pathlib.html#pathlib.Path.resolve) to ensure that regardless of how/where the script is called, a concrete absolute path is used.

Reply:
This might not be strictly necessary, but it's solved `Path` related edges in the past for me.